src/System.Net.Primitives/tests/UnitTests/CookieContainerTest.cs

```diff
@@ -198,14 +205,132 @@ public static IEnumerable<object[]> SetCookiesData()
                 }
             }; // Use escaped values (1)
 
-            yield return new object[] {
+            yield return new object[]
+            {
                 u6,
                 "name98=\"\"",
                 new Cookie[]
                 {
                     new Cookie("name98", "\"\"")
                 }
             }; // Use escaped values (2)
+            
+            yield return new object[]
+            {
+                u,
+                "locale=en, uuid=4b8b2dd7-d91a-49ee-80c6-8cb7df1fae46, country=US, _m_ask_fm_session=session1",
+                new Cookie[]
+                {
+                    new Cookie("locale", "en"),
+                    new Cookie("uuid", "4b8b2dd7-d91a-49ee-80c6-8cb7df1fae46"),
+                    new Cookie("country", "US"),
+                    new Cookie("_m_ask_fm_session", "session1")
```

Comment:
I think you should remove all of these references to ask fm.

Reply:
We had that discussion before. It's test data, any strings are ok.